web-server/v0.4/src/pages/Dashboard/Summary.js

```diff
@@ -574,7 +495,7 @@ class Summary extends ReactJS.Component {
             title={selectedResults[0]['run.name']}
             content={
               <Tag color="blue" key={selectedController}>
-                {'controller: ' + selectedController}
+                {`controller: ${  selectedController}`}
```

Comment:
Formatting?

Reply:
This is another feature of ES 6 (ECMAScript 6) that allows you to use template string literals instead of string concatenation. Template string literals should only be used where interpolation is happening or new lines are being created. I'll remove the space before `selectedController` so we can achieve a cleaner syntax here.